src/kirby/components/icon/custom-icon-settings.ts

```diff
@@ -1,8 +1,15 @@
 import { InjectionToken } from '@angular/core';
 
+export interface CustomIcon {
+  name: string;
+  svg?: string;
+  unicode: string;
+  fromCharCode?: string;
```

Comment:
Consider naming the property as is (the `fromCharCode` is the method used to generate the prop):
```suggestion
  charCode?: string;
```